.github/workflows/depviz.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: Doozers/depviz-action@e68e52e8761f640c9dd3bb3dd5859c7f4d29bbe5
```

Comment:
Can you use moul/depviz-action directly on this repo?

Reply:
Yes this is only for the example, will be replace by `moul/depviz-action@versionning`